delivery/services/staging_service.py

```diff
@@ -1,215 +0,0 @@
-
-import logging
-import threading
-import os
-import signal
-
-from delivery.models.db_models import StagingStatus
-from delivery.exceptions import RunfolderNotFoundException, InvalidStatusException
-
-log = logging.getLogger(__name__)
-
-
-class StagingService(object):
-    """
-    Starting in this context means copying a directory or file to a separate directory before delivering it.
-    This service handles that in a asynchronous way. Copying operations (right nwo powered by rsync) can be
-    started, and their status monitored by querying the underlying database for their status.
-    """
-
-    # TODO On initiation of a Staging service, restart any ongoing stagings
-    # since they should all have been killed.
-    # And if we do so we need to make sure that the Staging service
-    # acts as a singleton, look at:
```

Comment:
Good suggestion - will probably not do this now, but I'll add your suggestion to the comment section so that it's not lost once we fix this aspect.